### DISCUSSION TITLE
Help with wiring

### DISCUSSION THREAD

**Discussion by mrdavidscott** [aqualinkd/AqualinkD#279] [Q&A] [answered]
Hello, where does the brown wire and green wire of the RS-485 Connect to? I didn't see it in the instructions.

**Comment by sfeakes** [comment]
That depends on the connector you have.  But if using the one recommended in the wiki, then those wires will terminate the RS485 buss if needed.  Many cases you won’t want to terminate so don’t to anything with those wires, but sometimes you’ll need to terminate the connection, and to do that you simply connect to two together.  It explains this in the FTDI documentation for that adapter.
As for if you need to terminate, that will depend on what already on the buss and where you connect the adapter to the buss.

**Reply by mrdavidscott** [reply]
Hi and thanks for the quick reply. I'm using the connector recommended. I have the rs8 one touch and will be connecting to the plug in the house on the one touch panel. The one touch is the only thing on that buss.

**Reply by sfeakes** [reply, marked as answer]
You shouldn’t need to terminate the adapter in that setup.  You also only need to connect data+ and data- wires, so only 2 of the 6 wires.
